using-timescaledb/continuous-aggregates.md

```diff
@@ -72,6 +72,24 @@ hour. See the [`CREATE VIEW (Continuous Aggregate)`][api-continuous-aggs-create]
 section of our documentation for all of the options and requirements for the
 command.
 
+You can also create multiple continuous aggregates on the same raw hypertable. For example, we may
+want to query per-device readings aggregated on a daily basis. To do so, we can create a continuous
+aggregate as so:
```

Comment:
should this be `like so`?

Reply:
I used as so because that is what it was above, but I personally think as so is weird. I'll change both the like so.